editor/src/clj/editor/texture_set.clj

```diff
@@ -64,13 +64,28 @@
      :width width
      :height height}))
 
+(defn- flat-array->2d-points [flat-array]
+  (into [] (map vec (partition 2 flat-array))))
+
+(defn- ->anim-frame-from-geometry
+  [page-index frame-geometry]
+  {:page-index page-index
+   :tex-coords (flat-array->2d-points (:uvs frame-geometry))
+   :vertex-coords (flat-array->2d-points (:vertices frame-geometry))
```

Comment:
Apparently the functions requires the vertex data to be in this form:
[[p0x, p0y], [p1x, py1] ... ] and so on